src/sgame/sg_bot_ai.cpp

```diff
@@ -824,7 +824,7 @@ AINodeStatus_t BotActionFight( gentity_t *self, AIGenericNode_t *node )
 
 	//aliens have radar so they will always 'see' the enemy if they are in radar range
 	float goalDist = DistanceToGoalSquared( self );
-	if ( ( myTeam == TEAM_ALIENS && goalDist <= Square( ALIENSENSE_RANGE ) )
+	if ( ( myTeam == TEAM_ALIENS && goalDist <= Square( g_bot_aliensenseRange.Get() ) ) // cf ALIENSENSE_RANGE
```

Comment:
No merge commits please

Reply:
that was just to fix a conflict quickly. I don't like those merge/empty commits neither and will rebase instead.